### PR TITLE
Delivery Overview discard table improvements

### PR DESCRIPTION
**Pull request by forstisabella**
### Proposed changes
Added a table of all common delivery overview discard errors

### Merge timing
When Delivery Overview (Streaming Connections) goes GA

### Related issues (optional)
https://segment.atlassian.net/browse/DOC-842

### CODE REVIEW

File: src/connections/delivery-overview.md [hunk 1]
```diff
@@ -91,9 +91,17 @@ During the Delivery Overview beta, you can use the Event Delivery alerting featu
...
+Please note that this is dependent on your [notification settings](/docs/segment-app/#segment-settings). For example, if the threshold is set to 99%, then you'll be notified each time less than 100% of events fail. 
 
 
 ## Why is the Delivery Overview page only available for cloud-mode destinations? 
 Similar to Segment's [Event Delivery](/docs/connections/event-delivery/) feature, the Delivery Overview page is only available for server-side integrations (also known as cloud-mode destinations). You won't be able to use the Delivery Overview page for client side integrations (also known as device-mode destinations) because device-mode data is sent directly to the destination tool's API. In order to report on deliverability, data must be sent to destinations using a server-side connection. 
 
+## Troubleshooting
+
+The Delivery Overview pipeline steps Failed on Ingest, Filtered at Source, Filtered at Destination, and Failed Delivery display a [discard table](#discard-table) with information about why your events might have failed to reach their destination.
```
Comment (forstisabella) [inline src/connections/delivery-overview.md:102]:
```suggestion
The Delivery Overview pipeline steps Failed on Ingest, Filtered at Source, Filtered at Destination, and Failed Delivery display a [discard table](#discard-table) with information about why your events failed or were discarded.
```

**Comment by cmastr** [comment]
Will come back to review this

**Review by cmastr — APPROVED** [review]

File: src/_includes/content/delivery-overview-discards.html [hunk 2]
```diff
@@ -0,0 +1,613 @@
...
+<script>
+  function searchFilter() {
+    var input, filter, table, tr, samp, i, txtValue;
+
+    input = document.getElementById("filterInput");
+    filter = input.value.toUpperCase();
+    table = document.getElementById("settingsTable");
+    tr = document.getElementsByClassName("settingRow");
+    for (i = 0; i < tr.length; i++) {
+      samp = tr[i].getElementsByTagName("samp")[0];
```
Comment (ama0223) [inline src/_includes/content/delivery-overview-discards.html:594]: if you wanna filter by both columns, you'll prob need to wrap the Discard Reason <td> entries with a different tag but similar to how you did with <samp> then check for it in `txtValue` also. Or maybe assign it to a specific id like `<td id=discardTitle..` if that works

Comment (ama0223) [reply src/_includes/content/delivery-overview-discards.html:594]: you could try it on a few entries first to make sure something like that would work then apply a search + replace all

File: src/connections/sources/catalog/libraries/server/http-api/index.md [hunk 3]
```diff
@@ -476,6 +476,6 @@ When sending a HTTP call from a user's device, you can collect the IP address by
 
 1. Double check that you've set up the library correctly.
 
-2. Make sure that you're calling a Segment API method once the library is successfully installed—[`identify`](#identify), [`track`](#track), and so on.
+2. Make sure that you're calling a Segment API method once the library is successfully installed—[Identify](#identify), [Track](#track), and so on.
```
Comment (cmastr) [inline src/connections/sources/catalog/libraries/server/http-api/index.md:479]:
```suggestion
2. Make sure that you're calling a Segment API method after the library is successfully installed—[Identify](#identify), [Track](#track), and so on.
```

File: src/connections/sources/catalog/libraries/server/http-api/index.md [hunk 4]
```diff
@@ -320,7 +320,7 @@ The `alias` call has the following fields:
   {% include content/spec-field-user-id.md %}
 </table>
 
-For more details on the `alias` call and payload, check out our [Spec](/docs/connections/spec/alias/).
+For more details on the Alias call and payload, check out the [Segment Spec](/docs/connections/spec/alias/).
```
Comment (cmastr) [inline src/connections/sources/catalog/libraries/server/http-api/index.md:323]:
```suggestion
For more details on the Alias call and payload, see the [Segment Spec](/docs/connections/spec/alias/).
```

File: src/connections/delivery-overview.md [hunk 5]
```diff
@@ -91,9 +91,17 @@ During the Delivery Overview beta, you can use the Event Delivery alerting featu
 
 Once you enable alerts, if the successful delivery rate of all events is less than the threshold percentage in the last 24 hours, you'll be notified through in-app notification and/or workspace email. 
 
-Please note that this is dependent on your [notication settings](/docs/segment-app/#segment-settings). For example, if the threshold is set to 99%, then you'll be notified each time less than 100% of events fail. 
+Please note that this is dependent on your [notification settings](/docs/segment-app/#segment-settings). For example, if the threshold is set to 99%, then you'll be notified each time less than 100% of events fail.
```
Comment (cmastr) [inline src/connections/delivery-overview.md:94]:
```suggestion
Note that this is dependent on your [notification settings](/docs/segment-app/#segment-settings). For example, if the threshold is set to 99%, then you'll be notified each time less than 100% of events fail. 
```

File: src/connections/delivery-overview.md [hunk 6]
```diff
@@ -91,9 +91,17 @@ During the Delivery Overview beta, you can use the Event Delivery alerting featu
 
 Once you enable alerts, if the successful delivery rate of all events is less than the threshold percentage in the last 24 hours, you'll be notified through in-app notification and/or workspace email.
```
Comment (cmastr) [inline src/connections/delivery-overview.md:92]:
```suggestion
After you enable alerts, if the successful delivery rate of all events is less than the threshold percentage in the last 24 hours, you'll be notified through in-app notification and/or workspace email. 
```

File: src/_includes/content/retl-discards.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
+<table>
```
Comment (cmastr) [inline src/_includes/content/retl-discards.md:1]: Just for my own knowledge, do include tables have to be HTML instead of markdown?

Comment (forstisabella) [reply src/_includes/content/retl-discards.md:1]: no, they can be markdown, most tables on the site are. but with the search and filtering functionality this table had to be formatted with HTML

**Pull request merged**